src/app/chat/chat-list/chat-list.component.scss

```diff
@@ -84,6 +83,28 @@ clickable-item {
       width: fit-content;
     }
 
+    .indicator-container {
+      display: flex;
+      align-items: center;
+      flex-direction: row;
+      flex-wrap: wrap;
+      align-content: center;
+      ion-button {
+        cursor: default !important;
+        font-size: 11px;
+        margin: 0;
+        margin-right: 9px;
+        --padding-start: 0;
+        --padding-end: 0;
+        &.info {
+          font-size: 14px;
```

Comment:
better handle the font in `typography.scss` together

Reply:
@shawnm0705 this is custom for this component. and not even apply for other buttons in this component. i don't think we need to put in typography.scss.